src/index.ts

```diff
@@ -65,7 +65,7 @@ export class Auth {
   }
 
   // AuthorizeURL returns an
-  async AuthorizeURL (scopes: string[], state: string): Promise<string> {
+  async AuthorizeURL (scopes: string[], state: string, qrcode: boolean): Promise<string> {
```

Comment:
Let's define an options type to hold qrcode, as the optional third parameter. That is a bit more idiomatic JS.

```
interface AuthorizeOptions {
  qrcode: boolean
}
```